haskell/haskell.bzl

```diff
@@ -145,6 +145,10 @@ def _mk_binary_rule(**kwargs):
                 default = False,
                 doc = "Requires that the coverage metric is matched exactly, even doing better than expected is not allowed.",
             ),
+            "coverage_source_patterns": attr.string_list(
+                default = ["//..."],
```

Comment:
We don’t want to expose the pattern matching implementation as a stable feature yet, because we are hoping for upstream bazel to provide it. You should prefix anything that depends on it with `experimental_`.